src/lib/services/simulator.ts

```diff
@@ -39,39 +37,16 @@ function sleep(millliseconds: number): Promise<void> {
 export class SimulatorService implements ISimulatorService {
   private composeOptions: string
   private docker: Docker;
-  public simulatorLocation: string;
+  public location: string;
 
   constructor() {
-    this.simulatorLocation = "";
+    this.location = path.resolve(__dirname, '..');
```

Comment:
@epsjunior would this install the studio in the parent folder? can you please explain why this decision and why have we removed the location option?

Reply:
@cristiam86 we are not downloading studio repo anymore, just running the images already deployed by studio into docker containers, so we don't need a location to download the studio

the location var is being used to read the new .env file

.env file is in the parent folder of dist/index.js